src/lib/utils/embed.ts

```diff
@@ -27,8 +27,8 @@ export type APIEmbedWithoutDefault = Omit<APIEmbed, 'timestamp' | 'footer'>;
 export function success(description: string): APIEmbed {
 	return {
 		...defaultEmbed(),
-		title: `${BirthdayyEmojis.Success} Success`,
-		description: `${BirthdayyEmojis.ArrowRight} ${description}`,
+		title: `${Emojis.Success} Success`,
```

Comment:
Does this also need localisation?

Reply:
In the future yes I think it will be necessary